apis/swagger.yml

```diff
@@ -2447,6 +2470,15 @@ definitions:
         additionalProperties:
           type: "string"
 
+  ResizeOptions:
```

Comment:
If this is only used in daemon side, I do not think we should list this in swagger.yml.

Reply:
No, this struct is used by both

Reply:
I see. I found that you are not using this `ResizeOptions ` to communicate between client side and daemon side. You indeed use `ResizeOptions ` in package client, but when sending request, you split this struct into two separate parameters:

```
query.Set("h", strconv.Itoa(int(opts.Height)))
query.Set("w", strconv.Itoa(int(opts.Width)))
```

So in daemon side, you use both two parameters above to construct this struct `ResizeOptions` again.

I am afraid that the way is not the same as other struct used by both client and daemon side.

Correct me if I missed something, please.